src/jobflow_remote/remote/host/remote.py

```diff
@@ -323,6 +323,28 @@ def remove(self, path: str | Path) -> None:
         self._execute_remote_func(self.connection.sftp().remove, str(path))
 
     def rmtree(self, path: str | Path, raise_on_error: bool = False) -> bool:
+        """Recursively delete a directory tree on a host.
+
+        This method must be implemented by subclasses of  `BaseHost`.
```

Comment:
Same as for the local host, here its not anymore the baseclass so I guess this paragraph should be removed, maybe the first sentence could be "Recursively delete a directory tree on this remote host."